Makefile

```
@@ -119,7 +119,7 @@ source/fixtures/swagger.yaml/%.refract.parse-result.1.0.yaml: source/fixtures/sw
 	@$(FURY_YAML) $< > $@ || true
 
 node_modules:
-	@npm install --no-optional hercule dredd js-yaml media-typer chai fury-cli
+	@npm install --no-optional hercule dredd js-yaml media-typer chai @apielements/cli
```

```suggestion
	@npm install --no-optional
```

Since we have a lockfile now, would this work? (then we won't have to repeat the dependencies)